operator/builtin/input/tcp/tcp.go

```diff
@@ -53,6 +54,17 @@ func (c TCPInputConfig) Build(context operator.BuildContext) ([]operator.Operato
 		return nil, err
 	}
 
+	if c.MaxBufferSize < 0 || c.MaxBufferSize > 10 {
```

Comment:
What happens if the buffer size is set to 0?

Is 10 just an arbitrary value?

Reply:
If set to 0, we will default to 1. 

10 is arbitrary, but I feel that there should be some upper limit. If not 10, something reasonable.